### DISCUSSION TITLE
error saat run dashboard flask

### DISCUSSION THREAD

**Discussion by sarahoctav** [teamalgoritma/community#618] [Q&A] [answered]
saya sudah mengikuti video arahan dalam menjalankan app.py , tetapi terjadi error seperti ini kak👇🏻
<img width="1440" alt="Screenshot 2023-04-10 at 23 10 41" src="https://user-images.githubusercontent.com/75492575/230943491-fefdeba3-37d2-419c-94fa-9532157b2124.png">

**Comment by Yusraf002** [comment]
Dear @sarahoctav 
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak.

**Comment by fafilia** [comment]
Dear @sarahoctav ,
Mungkin bisa dicek di bagian log errornya Bu, adakah keterangan yang menunjukkan errornya pada baris kode secara spesifik? Jika berkenan mohon dilampirkan pesan errornya secara lengkap.

**Reply by sarahoctav** [reply, marked as answer]
<img width="1440" alt="Screenshot 2023-04-12 at 19 02 41" src="https://user-images.githubusercontent.com/75492575/231451091-1ad7d5ae-e1f8-427c-bb32-3a45bff968c4.png">
ini ka

**Reply by kevbow** [reply]
Halo @sarahoctav, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
